src/components/theme/FormattedDate/FormattedDate.jsx

```diff
@@ -13,17 +13,30 @@ const FormattedDate = ({
   relative,
   className,
   locale,
+  isHistory,
```

Comment:
This is plain wrong, do not change the default behavior of generic components please.

If the attribute `title` has a flaw, let's fix it.